### DISCUSSION TITLE
How do I Change from IP to hostname on accessing web console

### DISCUSSION THREAD

**Discussion by jozb09** [Security-Onion-Solutions/securityonion#11459] [2.4]
### Version

2.4.10

### Installation Method

Security Onion ISO image

### Description

configuration

### Installation Type

Distributed

### Location

on-prem with Internet access

### Hardware Specs

Exceeds minimum requirements

### CPU

16

### RAM

32

### Storage for /

293G

### Storage for /nsm

721G

### Network Traffic Collection

span port

### Network Traffic Speeds

1Gbps to 10Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

No, there are no failures

### Logs

No, there are no additional clues

### Detail

During the initial installation of the manager, I choose IP mode on the way to access the web console. 

Now, I want to change from IP to hostname. Kindly, help me on this to do it.

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to Security Onion 2.4.

**Comment by dougburks** [comment]
Have you checked the documentation?
https://docs.securityonion.net/en/2.4/url-base.html
